src/CONST.ts

```diff
@@ -2642,6 +2643,10 @@ const CONST = {
         HTTPS: 'https',
         PUSHER: 'pusher',
     },
+    EVENTS: {
+        SCROLLING: 'scrolling',
+    },
+    IS_DESKTOP_AND_WEB: Platform.OS === 'web' || Platform.OS === 'macos' || Platform.OS === 'windows',
```

Comment:
Is this really necessary, is it possible to use different files?
Note: Is this the abstraction we are talking in slack?

Reply:
Yes that's necessary as we want to call different `onScroll` method and also should `hoverable` component switch its behaviour ( for eg, show hover only once scroll stops ).

The reason we can't have different files is that we have the same implementation for web and desktop. So this flag tells us that whether we are on web or desktop.

Given that I should rename this key to `IS_DESKTOP_OR_WEB`.  👍🏻

> Note: Is this the abstraction we are talking in slack?

No. Abstraction is related to Event Emitters.

Reply:
Given the issue urgency. Its fine with me to take it up in separate followup PR as well to have separate files and not have platform check.

cc: @mountiny If you are fine as well.

Reply:
Okay. So the two components which will go to platform specific files are `BaseInvertedFlatList` and `Hoverable`. The `Hoverable` component is already for web and desktop, since we have `index.native.ts` there. So we only need to refactor `BaseInvertedFlatList` and some parts of `Hoverable`.

Reply:
We do prefer the different files per platform, @hurali97 are you saying here that this is not possible here?